src/server/__tests__/schema.test.js

```diff
@@ -44,7 +44,8 @@ describe('Schema', () => {
       _mapping: Mapping
     }`;
   test('could create query schemas', async () => {
-    const querySchema = getQuerySchema(config.esConfig);
+    await esInstance.initialize();
+    const querySchema = getQuerySchema(config.esConfig, esInstance);
     expect(removeSpacesNewlinesAndDescriptions(querySchema))
```

Comment:
Could you also update this file once you add fake nested data to `commands.sh` file? so that this will check generated schema is correct. thanks!